### PR TITLE
Expose to clearnet through Cloudflare argo tunnel

### LINKED ISSUE #1147: New Doc ; Expose to clearnet using Cloudflare Argo Tunnel
Started working on docs for  : https://github.com/btcpayserver/btcpayserver-docker/pull/667

Trying to finish first version asap.

### PR DESCRIPTION
**Pull request by Bangalisch**
Made a first write-up of the expose to clearnet via Cloudflare PR https://github.com/btcpayserver/btcpayserver-docker/pull/667

cc @NicolasDorier @danielcharrua for review and additions and maybe relocate it? 

Closes #1147

### CODE REVIEW

**Comment by NicolasDorier** [comment]
Very useful, but this is not the right category.
This isn't only relative to manual deployment.

I really like the additional instructions and screenshot, however, I would prefer you modify this page https://github.com/btcpayserver/btcpayserver-docker/blob/master/docs/cloudflare-tunnel.md directly.

Note that some instruction are different (you forgot `[[ "$REVERSEPROXY_DEFAULT_HOST" ]] && REVERSEPROXY_DEFAULT_HOST="$BTCPAY_HOST"` for example).

I think @dennisreimann has a system that can grab the documentation from the btcpayserver-docker and automatically build our doc from it. I will ask him.

File: docs/Deployment/ManualDeployment.md [hunk 1]
```diff
@@ -125,28 +125,30 @@ You need to create an account on Cloudflare and add their DNS to [your domain](h
 
 ### Step 2 Setting up Argo Tunnel
 
-After you've added the DNS and is propagated, you can start the setup of the Argo tunnel. Go to [Zero Trust](https://dash.teams.cloudflare.com/c1cd2ab2129af8897457d6f97c082725/onboarding) option on the left menu, go to "access", then click "tunnels".
+After you've added the DNS and is propagated, you can start the setup of the Argo tunnel. 
+Go to [Zero Trust](https://dash.teams.cloudflare.com/c1cd2ab2129af8897457d6f97c082725/onboarding) option on the left menu, go to "access", then click "tunnels".
```
Comment (danielcharrua) [inline docs/Deployment/ManualDeployment.md:129]: Guys there is a link to an account that is wrong, the string after https://dash.teams.cloudflare.com/ is identifying an account, please remove that.

**Review by danielcharrua — CHANGES_REQUESTED** [review]

**Comment by dennisreimann** [comment]
Nice, one it's moved over to the Docker repo, we can use this PR to simply integrate it here. I can take care of that after it has been updated and added in the Docker repo.

File: docs/Deployment/ManualDeployment.md (hunk 1 above)
Comment (Bangalisch) [reply docs/Deployment/ManualDeployment.md]: Took this out in last update :) TY!

**Comment by Bangalisch** [comment]
> Nice, one it's moved over to the Docker repo, we can use this PR to simply integrate it here. I can take care of that after it has been updated and added in the Docker repo.

@dennisreimann Changed location of it here to FAQ - Deployments + shortened a little further so is more FAQ fit.
@NicolasDorier Updated script to reflect btcpay docker docs :)

**Comment by NicolasDorier** [comment]
@Bangalisch no, what I am saying is that rather than writing the doc from scratch here, I would prefer you edit this one https://github.com/btcpayserver/btcpayserver-docker/blob/master/docs/cloudflare-tunnel.md

With the improvement you made.

btcpayserver-doc is then smart enough to update the doc website by pulling out the doc for the btcpayserver-docker repo.

**Comment by Bangalisch** [comment]
@NicolasDorier did that too ;) https://github.com/btcpayserver/btcpayserver-docker/pull/669

**Comment by dennisreimann** [comment]
Ok, now that the changes in the docker repo have been made ­­­— where shall we integrate them in the docs website?

**Comment by danielcharrua** [comment]
maybe we need to add another page below Hardware deployment? https://docs.btcpayserver.org/Deployment/Hardware/

Is where there are the other methods of exposing to clearnet, dynamic DNS, reverse proxy to tor, etc..

What do you think? @NicolasDorier

**Comment by NicolasDorier** [comment]
@danielcharrua I agree, I will make a separate PR, I want to re-organize the Hardware section

**Comment by NicolasDorier** [comment]
I will open a new PR as @Bangalisch already ported his doc to the docker repo, the only thing missing is reorganizing the hardware section.

**Pull request closed without merging**